.github/workflows/quality.yml

```diff
@@ -157,10 +157,14 @@ jobs:
         python-version: "3.10"
         architecture: x64
     - name: Install dependencies
-      run: python -m pip install -U pip black pyupgrade isort
+      run: |
+        python -m pip install -U pip black pyupgrade isort
+        sudo apt install -y clang-tidy
     - name: Check Python Formatting
       run: black --check .
     - name: Check Python import sorting
       run: isort --check .
     - name: Check pyupgrade
       run: pyupgrade --py37-plus src/picologging/*.py tests/**/*.py
+    - name: Check clang-tidy
+      run: clang-tidy src/picologging/*.cxx -- -I/usr/include/python3.10/ -std=c++17
```

Comment:
This works locally but pipelines require approval so not sure if arguments need changing.